docs/Arena-Object-Definitions.md

```diff
@@ -0,0 +1,346 @@
+# Animal-AI Environment Objects
```

Comment:
If everything in this repo is now docs, it's probably worth getting rid of the `docs` folder and just having stuff at the top level

Reply:
This repo is also responsible for publishing releases of AAI. It's important to note that eventually we ill merge all three repos and still use animal-ai as the name of the repo, for all purposes. Changing to animal-ai-docs would be redundant now and in the future.